src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala

```diff
@@ -190,12 +190,13 @@ trait HeadersProcessor extends ScorexLogging {
       Failure(new Error("Parent header is marked as semantically invalid"))
     } else {
       Success()
-    }.recoverWith { case thr =>
```

Comment:
What is the purpose of this change?

Reply:
I removed it, cause this change is out of scope of current task. I think that this `recoverWith` will be only applied to `Success` and hence never will be called.

Here is link to small example that demonstrates that `recover` in `validate` won't work, rather than in `validateF` [Scastie](https://scastie.scala-lang.org/uFFsJoxfQqu6isDhZinrjg)